pyi.py

```diff
@@ -146,6 +152,12 @@ def run_check(self, plugin, **kwargs):
 class PyiVisitor(ast.NodeVisitor):
     filename: Path = Path("(none)")
     errors: list[Error] = field(default_factory=list)
+    # Mapping of all private TypeVars/ParamSpecs/TypeVarTuples to the nodes where they're defined
+    typevarlike_defs: dict[TypeVarInfo, ast.Assign] = field(default_factory=dict)
+    # Mapping of each name in the file to the no. of occurrences
+    all_name_occurrences: Counter[str] = field(default_factory=Counter)
+    # Collection of the linenos of all the union statements we've seen
+    unions_already_seen: set[int] = field(default_factory=set)
```

Comment:
```suggestion
```
Doesn't look like we're using this

Reply:
Yeah, was left over from a previous attempt at fixing https://github.com/PyCQA/flake8-pyi/pull/83#issuecomment-1013897332 before I implemented Akuli's suggestion. Thanks!